### PR TITLE
Access to the subscription whitelist button

### LINKED ISSUE #1952: Revert fake door and change the copy
Revert this one https://github.com/ChainSafe/ui-monorepo/pull/1937
It should be along the line: "Want to be a beta tester?"
And then tell our users to go to discord, and we should add them from there.

### PR DESCRIPTION
**Pull request by Tbaut**
closes https://github.com/ChainSafe/ui-monorepo/issues/1952

---
Submission checklist: 

   

#### Layout
- [x] Change looks good in the desktop web ui
- [x] Change looks good in the mobile web ui

#### Theme
- [x] Components / elements inspected in light mode 
- [x] Components / elements inspected in dark mode

### CODE REVIEW

File: packages/files-ui/src/Components/Elements/BetaModal.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      maxWidth="sm"
+      onClose={onHide}
+      mobileStickyBottom={false}
+    >
+      <div className={classes.root}>
+        <Typography
+          variant="h2"
+          className={classes.title}
+        >
+          <Trans>Join the beta</Trans>
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Elements/BetaModal.tsx]:
```suggestion
          <Trans>Need more storage?</Trans>
```

File: packages/files-ui/src/Components/Elements/BetaModal.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        </Typography>
+        <Typography variant="h4">
+          <Trans>Join our beta testers by requesting access in the &quot;Files-support&quot; channel on Discord.</Trans>
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Elements/BetaModal.tsx]:
```suggestion
          <Trans>We are rolling out our subscription plans to a limited number of users. Request access to be able to upgrade to a plan with more storage.</Trans>
```

File: packages/files-ui/src/Components/Elements/BetaModal.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        </Typography>
+        <div className={classes.buttonContainer}>
+          <Button
+            data-posthog="Beta-test-discord"
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Elements/BetaModal.tsx]:
```suggestion
            data-posthog="Subscription-whitelist-form"
```

File: packages/files-ui/src/Components/Layouts/AppHeader.tsx [hunk 4]
```diff
@@ -184,6 +190,14 @@ const AppHeader = ({ navOpen, setNavOpen }: IAppHeader) => {
                 >
                   <Trans>Report a bug</Trans>
                 </Button>
+                <Button
+                  data-posthog="Join-beta"
+                  variant="tertiary"
+                  size="small"
+                  onClick={onJoinBetaClick}
+                >
+                  <Trans>Get more storage</Trans>
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Layouts/AppHeader.tsx]:
```suggestion
                  <Trans>Need more storage?</Trans>
```

**Review by asnaith — APPROVED** [review]

**Review by FSM1 — APPROVED** [review]

File: packages/files-ui/src/Components/FilesRoutes.tsx [hunk 5]
```diff
@@ -43,7 +43,8 @@ export const ROUTE_LINKS = {
     const adjustedRawCurrentPath = !rawCurrentPath ? "/" : rawCurrentPath
     return `/shared/${bucketId}${adjustedRawCurrentPath}`
   },
-  DiscordInvite: "https://discord.gg/zAEY37fNb2"
+  DiscordInvite: "https://discord.gg/zAEY37fNb2",
+  SubscriptionWhitelistForm: ""
```
Comment (tanmoyAtb) [inline packages/files-ui/src/Components/FilesRoutes.tsx]:
```suggestion
  DiscordInvite: "https://discord.gg/zAEY37fNb2"
```

Comment (tanmoyAtb) [inline packages/files-ui/src/Components/FilesRoutes.tsx:47]: Don't think this is needed.
```suggestion
  DiscordInvite: "https://discord.gg/zAEY37fNb2"
```

**Review by tanmoyAtb — APPROVED** [review]
Looks good

File: packages/files-ui/src/Components/FilesRoutes.tsx (hunk 5 above)
Comment (Tbaut) [reply packages/files-ui/src/Components/FilesRoutes.tsx]: This needs to be updated before merging. @sweetpea22 will give me the link for it today I believe

Comment (tanmoyAtb) [reply packages/files-ui/src/Components/FilesRoutes.tsx]: Ahh ok 👍

File: packages/files-ui/src/Components/Elements/BetaModal.tsx [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+        <Typography
+          variant="h2"
+          className={classes.title}
+        >
+          <Trans>Need more storage?</Trans>
+        </Typography>
+        <Typography variant="h4">
+          <Trans>
+            We are rolling out our subscription plans to a limited number of users.
+            Request access to be able to upgrade to a plan with more storage.
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Elements/BetaModal.tsx]:
```suggestion
            Join our new limited-access subscription plans to upgrade to a plan with more storage.
```

**Comment by sweetpea22** [comment]
https://chainsafe.typeform.com/to/unZ0veao

what do we think of this?

File: packages/files-ui/src/Components/FilesRoutes.tsx (hunk 5 above)
Comment (Tbaut) [reply packages/files-ui/src/Components/FilesRoutes.tsx]:
```suggestion
  DiscordInvite: "https://discord.gg/zAEY37fNb2",
  SubscriptionWhitelistForm: "https://chainsafe.typeform.com/to/unZ0veao"
```

**Pull request merged**